javatests/arcs/core/analysis/OutcomeTest.kt

```diff
@@ -0,0 +1,41 @@
+package arcs.core.analysis
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.fail
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [Outcome]. */
+@RunWith(JUnit4::class)
+class OutcomeTest {
+    val success = Outcome.Success<Int>(10)
+    val failure = Outcome.Failure<Int>("Failure Example")
```

Comment:
I think it'd be better to inline these in the tests below. That would make them more readable:

e.g.
```
asssertThat(Outcome.Success(10).getOrElse { 20 }).isEqualTo(10)
```

Reply:
Good idea. Done.